tests/MatrixDotNet.UnitTesting/MatrixTests/StatisticsTests/MatrixMeanTest.cs

```diff
@@ -6,7 +6,7 @@ namespace MatrixDotNetTests.MatrixTests.StatisticsTests
 {
     public class MatrixMeanTest
     {
-        [Fact]
+        //[Fact]
```

Comment:
¿?¿

Reply:
Fuck, I have branch older than you pushed to master yesterday, thus my tests not passed with error `InvalidOperationException`.